drivers/amazonec2/amazonec2.go

```diff
@@ -199,9 +205,13 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 	d.SwarmMaster = flags.Bool("swarm-master")
 	d.SwarmHost = flags.String("swarm-host")
 	d.SwarmDiscovery = flags.String("swarm-discovery")
-	d.SSHUser = "ubuntu"
+	d.SSHUser = flags.String("amazonec2-ssh-user")
 	d.SSHPort = 22
 
+	if d.SSHUser == "" {
```

Comment:
This isn't needed as the cli.StringFlag will always be set to "ubuntu" as the default.